test/unit/main/post/views/filters/active-filters.directive.spec.js

```diff
@@ -1,81 +0,0 @@
-describe('post active filters directive', function () {
```

Comment:
this test is part of the things that are gone with the refactor because we don't have the directive anymore